src/libs/intents/localMethods.ts

```diff
@@ -34,8 +36,9 @@ export const asyncLogout = async (client?: CozyClient): Promise<null> => {
 
 // Since logout is used from localMethods
 // it can't be async for now.
-const logout = (client?: CozyClient): (() => Promise<null>) => {
-  return async () => asyncLogout(client)
+const logout = async (client?: CozyClient): Promise<null> => {
+  await asyncLogout(client)
+  return null
```

Comment:
Why do we need a `return null` here? Doesn't `asyncLogout` already do this?

Reply:
Correct, fixed